audb/core/dependencies.py

```diff
@@ -563,6 +577,8 @@ def error_message_missing_object(
     else:
         object_name = object_type
 
+    # TODO: check if we can combine the two error messages
```

Comment:
Do you want to check this now, or later?

Reply:
It's not really related to this PR, I was just wondering why we have this two cases. Can be tackled in a separate PR if we think it makes sense to have just a single case.

Reply:
We have to cases as one is for matching a single string as this can be search pattern, and the other one when a list with objects is given in which case we match against the actual names.

Reply:
Ah now I remember what confused me. To differentiate between the to cases we check if `missing_object_id` is a string or a list. But hen we only show the first element of the list and when I checked the cases where we call the function, we always put single elements into a list. So I guess it would make more sense to split into to different error messages or add an argument `is_pattern`.

Reply:
I created https://github.com/audeering/audb/issues/280.

And would propose to remove the comment here:

```suggestion

```

Reply:
done